### PR TITLE
Add benchmarks to parse,compile,plan,eval for 1,15,30 `OR`ed `LIKE`s

### PR DESCRIPTION
**Pull request by jpschorr**
Tests ported from https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/src/jmh/kotlin/org/partiql/jmh/benchmarks/MultipleLikeBenchmark.kt

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql/benches/bench_eval_multi_like.rs [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+
+    let mut rng = rand::rngs::StdRng::from_seed([42; 32]);
+    use rand::distributions::Distribution;
+    let chars = rand::distributions::Alphanumeric;
+    let random_size = rand::distributions::uniform::Uniform::from(5..=100);
+
+    let employee_data: Vec<Value> = combined
+        .enumerate()
+        .map(|(id, person)| {
+            //
```
Comment (alancai98) [inline partiql/benches/bench_eval_multi_like.rs:237]: Empty comment

```suggestion
```

File: partiql/benches/bench_eval_multi_like.rs [hunk 2]
```diff
@@ -0,0 +1,425 @@
+use std::time::Duration;
```
Comment (am357) [inline partiql/benches/bench_eval_multi_like.rs:1]: Maybe a short description of the bench (and the query it benchmarks can also help like `QUERY_15` or `QUERY_30`).

**Review by am357 — APPROVED** [review]
+1 on using `partiql` for benches.

File: partiql/Cargo.toml [hunk 3]
```diff
@@ -20,6 +20,25 @@ edition.workspace = true
...
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
+partiql-parser = { path = "../partiql-parser" }
+partiql-ast = { path = "../partiql-ast" }
+partiql-value = { path = "../partiql-value" }
+partiql-logical = { path = "../partiql-logical" }
+partiql-logical-planner = { path = "../partiql-logical-planner" }
+partiql-eval = { path = "../partiql-eval" }
```
Comment (alancai98) [inline partiql/Cargo.toml:29]: May need to add version numbers. For the 0.2.0 (https://github.com/partiql/partiql-lang-rust/pull/272/files), I wasn't able to publish without adding the version numbers as part of the dependencies.

File: partiql/benches/bench_eval_multi_like.rs [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+
+use partiql_eval::env::basic::MapBindings;
+use partiql_eval::eval::EvalPlan;
+use partiql_eval::plan::EvaluatorPlanner;
+use partiql_logical::{BindingsOp, LogicalPlan};
+use partiql_parser;
+use partiql_parser::{Parser, ParserResult};
+use partiql_value::{partiql_tuple, Bag, Tuple, Value};
+
+fn employee_data() -> Vec<Value> {
```
Comment (alancai98) [inline partiql/benches/bench_eval_multi_like.rs:15]: Should we also add an entry to the CHANGELOG for this new benchmark?

**Pull request merged**